nbodykit/cosmology/power/galaxy.py

```diff
@@ -0,0 +1,128 @@
+import numpy
+from . import transfers
+from ..cosmology import Cosmology
+from .linear import LinearPower
+
+class GalaxyPower(object):
+    """
+    An object to compute the galaxy/redshift power spectrum and related quantities,
+    using a transfer function from the CLASS code or the analytic
+    Eisenstein & Hu approximation.
+
+    Parameters
+    ----------
+    cosmo : :class:`Cosmology`, astropy.cosmology.FLRW
+        the cosmology instance; astropy cosmology objects are automatically
+        converted to the proper type
+    redshift : float
+        the redshift of the power spectrum
+    transfer : str, optional
+        string specifying the transfer function to use; one of
+        'CLASS', 'EisensteinHu', 'NoWiggleEisensteinHu'
+    b0 : float
+        the linear bias of the galaxy in a gaussian field
+    fnl : float
+        the non-gaussian parameter
+    p : float
+        p takes a between 1 and 1.6. 
+        p=1 for objects populating a fair sample of all the halos
+        p=1.6 for objects that populate only recently merged halos
+    Omega_m : float
+        the matter density parameter
+    H0 : float
+        the Hubble constant (in units of km/s Mpc/h)
+    c : float
+        speed of light (in units of km/s)
+        
+
+    Attributes
+    ----------
+    cosmo : class:`Cosmology`
+        the object giving the cosmological parameters
+    sigma8 : float
+        the z=0 amplitude of matter fluctuations
+    redshift : float
+        the redshift to compute the power at
+    transfer : str
+        the type of transfer function used
+    """
+    
+    def __init__(self, cosmo, redshift ,b0 ,fNL ,p ,Omega_m ,H0=73.8 ,c=3e5 ,transfer='CLASS'):
```

Comment:
Why not using cosmo.h and cosmo.Omega_m?

Reply:
Oh I completely missed that. That would be so much better. Thanks. Any other recommendations?